core/templates/pages/blog-dashboard-page/blog-dashboard-page-root.component.html

```diff
@@ -0,0 +1,20 @@
+<oppia-angular-root>
+  <div ng-controller="Base" dir="<[direction]>">
```

Comment:
Do we need ng-controller here?

Reply:
Same for `dir`, can you please check whether dir is handled inside `oppia-angular-root` and remove it?

Reply:
Done, Removed ng-controller and dir